source/globalCommands.py

```diff
@@ -1568,6 +1568,16 @@ def script_navigatorObject_devInfo(self,gesture):
 	script_navigatorObject_devInfo.__doc__ = _("Logs information about the current navigator object which is useful to developers and activates the log viewer so the information can be examined.")
 	script_navigatorObject_devInfo.category=SCRCAT_TOOLS
 
+	@script(
+		# Translators: Input help mode message for Open configuration directory command.
+		description=_("Opens NVDA configuration directory."),

```

Comment:
```suggestion
		description=_("Opens NVDA user configuration directory."),
```

Reply:
I second this suggestion. I think it's helpful to mention this is the user config directory.